app/uk/gov/hmrc/pillar2externalteststub/models/uktr/UktrSubmission.scala

```diff
@@ -30,10 +33,27 @@ trait UktrSubmission {
 }
 
 object UktrSubmission {
+  val UKTR_STUB_PROCESSING_DATE = "2022-01-31T09:26:17Z"
```

Comment:
why is this date static?

Reply:
This was done to follow the convention used elsewhere in the stubs: a statis date+time is put into the response, and this date+time is then cross-checked in the unit test. I can remove this, and populate the response with the actual date/time now, if you think that makes more sense.

Reply:
New commit added to set stub processing date/timezone to current date/timezone for LiableEntity tests.